cmd/gpu-kubelet-plugin/checkpoint.go

```diff
@@ -12,14 +12,14 @@ type Checkpoint struct {
 }
 
 type CheckpointV1 struct {
-	PreparedClaims PreparedClaims `json:"preparedClaims,omitempty"`
+	PreparedDevicesByClaimUID PreparedDevicesByClaimUID `json:"PreparedDevicesByClaimUID,omitempty"`
 }
 
 func newCheckpoint() *Checkpoint {
 	pc := &Checkpoint{
 		Checksum: 0,
 		V1: &CheckpointV1{
-			PreparedClaims: make(PreparedClaims),
+			PreparedDevicesByClaimUID: make(PreparedDevicesByClaimUID),
```

Comment:
for the future: make gpu plugin and CD plugin code paths symmetric w.r.t. which data to checkpoint, and which structures to use during unprepare